### PR TITLE
Removing some names

### PR DESCRIPTION
**Pull request by slyubomirsky**
Largely the work of `grep`. Will add more to this PR as I go on

I am not tackling renaming DEVOID itself (outside of comments/the readme) because that would require modifications to Coq code

(For fun/checking, I grepped for the following using `-r *` from the project root: `talia`, `ringer`, `uwplse`, `plse`, `washington`, `cse`, `PUMPKIN`, `DEVOID`, `ITP`)

### CODE REVIEW

**Comment by slyubomirsky** [comment]
On line 34 of README.md, there are references to some dependencies hosted on `uwplse`. Those are probably deanonymizing and should probably be built into your code submission in some other way. Not sure what to do about them but I did not want to note that explicitly here

**Comment by slyubomirsky** [comment]
`Assumptions.v` contains references to github issue numbers. Not sure if that's something you intend to change, as it's not deanonymizing, but I also wanted to point that out.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Talia Ringer, Nate Yazdani
+Copyright (c) 2019 ...
```
Comment (slyubomirsky) [inline LICENSE:3]: Lol! My grepping missed that! Will always check upper- and lowercase in the future

File: plugin/src/options.ml [hunk 2]
```diff
@@ -38,8 +38,8 @@ let is_search_equiv () = !opt_search_equiv
 let opt_smart_elim = ref (false)
 let _ = Goptions.declare_bool_option {
   Goptions.optdepr = false;
-  Goptions.optname = "Generate useful eliminators for DEVOID";
-  Goptions.optkey = ["DEVOID"; "search"; "smart"; "eliminators"];
```
Comment (tlringer) [inline plugin/src/options.ml:42]: Ah, if this one changes, all of the Coq files will have to change :)

**Pull request closed without merging**